sktime/transformations/series/theta.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+"""copyright: sktime developers, BSD-3-Clause License (see LICENSE file)."""
+
+__author__ = ["Markus Löning"]
+# __all__ = ["ThetaLinesTransformer"]
+
+import numpy as np
+import pandas as pd
+
+from sktime.transformations.base import _SeriesToSeriesTransformer
+from sktime.forecasting.base import ForecastingHorizon
+from sktime.forecasting.trend import PolynomialTrendForecaster
+
+
+class ThetaLines(_SeriesToSeriesTransformer):
+    """Decompose the original data into two or more ThetaLines."""
+
+    def __init__(self, theta):
+        self.theta = theta
+        super(ThetaLines, self).__init__()
+
+    def transform(self, Z, X=None):
+        """Transform data.
+
+        Parameters
+        ----------
+        Z : pd.Series
+            Series to transform.
+        X : pd.DataFrame, optional (default=None)
+            Exogenous data used in transformation.
+
+        Returns
+        -------
+        pd.DataFrame
+            Theta lines[1].
```

Comment:
maybe worth including more detail for returns part of docstring:
* name of variable that is returned
* how the shape of the dataframe is determined by the inputs. Something like number of rows = len(Z) and number of columns = len(theta).